contracts/JBveBanny.sol

```diff
@@ -355,9 +370,10 @@ contract JBveBanny is ERC721Votes, ERC721Enumerable, Ownable, ReentrancyGuard, J
 
     // Redeem the locked tokens to reclaim treasury funds.
     uint256 _reclaimedAmount = _terminal.redeemTokensOf(
-      _owner,
+      address(this),
```

Comment:
This contract holds the user's tokens right?

Reply:
oh, nice. ok then disregard the previous comment. We need a permissions check, but maybe use a modifier?